csharp/src/Microsoft.ML.OnnxRuntime/Tensors/ArrayTensorExtensions.shared.cs

```diff
@@ -27,7 +27,12 @@ public static class ArrayTensorExtensions
         /// <returns>A 1-dimensional DenseTensor&lt;T&gt; with the same length and content as <paramref name="array"/>.</returns>
         public static DenseTensor<T> ToTensor<T>(this T[] array)
         {
-            return new DenseTensor<T>(array);
```

Comment:
Just curious, can all these copy logics here be part of the DenseTensor ctor?

Reply:
I believe DenseTensor was originally copied from a .net library so doing it this way preserved that interface and would make future updates cleaner.